cmd/fyne/internal/commands/build.go

```diff
@@ -347,3 +348,24 @@ func targetOS() string {
 
 	return runtime.GOOS
 }
+
+func appendEnv(env *[]string, varName, value string) {
+	for i := range *env {
+		keyValue := strings.Split((*env)[i], "=")
+
+		if len(keyValue) >= 2 {
+			if keyValue[0] == varName {
+				keyValue[len(keyValue)-1] += " " + value
+
+				(*env)[i] = keyValue[0]
+				for _, v := range keyValue[1:] {
```

Comment:
This loop is a little hard to read - would it not be cleaner using `strings.SplitN` or similar to only split on the first = sign?